.circleci/config.yml

```diff
@@ -94,6 +94,13 @@ jobs:
             DOCKER_BUILD_ARGUMENTS="--cache-from securedrop-test-focal-py3:${fromtag:-latest}" securedrop/bin/dev-shell \
                   bash -c "/opt/venvs/securedrop-app-code/bin/pip3 install --require-hashes -r requirements/python3/develop-requirements.txt && make -C .. lint"
 
+      - run:
+          name: Check that translation machine objects are reproducible
```

Comment:
Why does this need to be its own step? Can it not be part of the `make lint` target?

Reply:
I'd originally assumed it would be a step in `translation-tests`.  But it doesn't benefit from `translation-tests`'s parallelism, since we might as well verify all languages at once, not just supported languages one at a time.

The `lint` job seemed like the next best logical place.  But I take `make lint` itself to be analogous to spellcheck for human-modified files, not machine-generated ones.  And I don't see any benefit to running `make verify-mo` locally at all.  I resisted the temptation to move it into its own job altogether.  ;-)

Would you prefer to see this run somewhere else?  See also some extra CI bits in #6954.

Reply:
It's fast enough that the overhead of re-installing the pip dependencies takes more time than actually running the verification script. I think merging into lint is probably the right option, I'll roll that into my upcoming PR.

Philosophically, I think it doesn't matter what is being linted/tested, rather it's that "lint" is all the fast tools that just analyze code while "test" is slower stuff that does execution, etc.

Reply:
Works for me!